roles/DCOS.master/tasks/main.yml

```diff
@@ -49,3 +56,8 @@
   when:
     - ansible_local is defined and ansible_local.dcos_installation is defined
     - (ansible_local.dcos_installation['version'] != dcos['version']) or (dcos['image_commit'] is defined and ansible_local.dcos_installation['dcos-image-commit'] != dcos['image_commit']) or latest_dcos_config_setup_is_installed.rc != 0
+
+- name: Remove CA service PSK
+  file:
```

Comment:
the file got just create this resource will never reach a OK state

Reply:
The psk is only needed during install. This is the desired end state.